cli/ssh.go

```diff
@@ -477,6 +482,12 @@ func (r *RootCmd) ssh() *serpent.Command {
 			Description: "Specifies whether to emit SSH output over stdin/stdout.",
 			Value:       serpent.BoolOf(&stdio),
 		},
+		{
+			Flag:        "ssh-host-prefix",
+			Env:         "CODER_CONFIGSSH_SSH_HOST_PREFIX",
+			Description: "Strip this prefix from the provided hostname to determine the workspace name.",
```

Comment:
```suggestion
			Description: "Strip this prefix from the provided hostname to determine the workspace name. This is useful when used as part of an OpenSSH proxy command.",
```

Not sure what we should write exactly, but I would like to make it clearer this is not a useful flag when using `coder ssh` directly.